lacci/lib/shoes/widgets/progress.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Shoes
+  class Progress < Shoes::Widget
```

Comment:
Does it need a display property or something for % progress?

Reply:
yeah it does i guess there is value and max as two proerties and
 i need to change this html into render method
 so i made this into draft

Reply:
Ah, sorry, this says draft. So you're probably already on it. Looks like "fraction" is the Shoes style for it, and it's a float between 0.0 and 1.0.